lib/providers/filesystem/index.js

```diff
@@ -42,7 +42,7 @@ function FileSystemProvider(options) {
 var namePattern = new RegExp('[^' + path.sep + '/]+');
 
 function validateName(name, cb) {
-  if (!name) {
+  if (!name || name.indexOf('..') !== -1) {
```

Comment:
This is incorrect. `..` as _part_ of a path component is valid:
```
/tmp/foo/from..to
/tmp/..valid-but-badlynamed
/tmp/.../dont-actually-use-paths-like-even-though-you-can
```

The last 2 examples above would be _hidden_ by UNIX dot-file logic, but that's more about `ls` defaults than anything else.